server/controllers/events/events_controller.go

```diff
@@ -627,9 +627,11 @@ func (e *VCSEventsController) HandleAzureDevopsPullRequestCommentedEvent(w http.
 		return
 	}
 
-	if *resource.Comment.IsDeleted {
-		e.respond(w, logging.Debug, http.StatusOK, "Ignoring comment event since it is linked to deleting a pull request comment; %s", azuredevopsReqID)
-		return
+	if resource.Comment.IsDeleted != nil {
```

Comment:
Do we need to use GetIsDeleted here too?

Reply:
Yep, once the other changes are pushed it will conflict and I'll rewrite this one a bit.

Reply:
@nitrocode - This is now caught up with #2838 so if people are happy with the content (still feels a bit weird targeting the URL) then it's good to go 😃